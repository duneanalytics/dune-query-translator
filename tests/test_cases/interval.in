select
  evt_block_time + interval '1 days' * term as can_claim_day,
  term,
  rank
from
  xen_ethereum.XENCrypto_evt_RankClaimed