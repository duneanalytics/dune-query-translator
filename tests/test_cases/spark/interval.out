/* Success! If you're still running into issues, check out https://dune.com/docs/query/syntax-differences/ or reach out in the #dune-sql Discord channel. */

SELECT
  evt_block_time + INTERVAL '1' day * term AS can_claim_day,
  term,
  rank
FROM xen_ethereum.XENCrypto_evt_RankClaimed